Update Docs - Latest, Next Versions Sync

sunilarjun opened the pull request:
This PR syncs v1.6 product and community v1.6. It also adds the Next docs version folder and Latest/v1.7  docs versions with the respective v1.7 docs content updates. The playbook files are also updated to now point to Latest, and I will follow up with an articles listing page update to point to Latest.

To further clarify, the Next version is based on the up-to-date v1.6 and has been synced with the community Next version for v1.7 content updates. 

The v1.7 docs version is based on the completely synced and up-to-date Next version.

Below is a reference list of sync commits and the relevant community PR to compare against when reviewing. Please let me know if any clarification is needed, thank you.

v1.6 related commits:
Reference commit - Reference PR:
https://github.com/rancher/elemental-product-docs/pull/43/commits/f7ceb023e391ffb6a946bfd789973c5083a32c70 - https://github.com/rancher/elemental-docs/pull/375
https://github.com/rancher/elemental-product-docs/pull/43/commits/d872d9777be221af3695a80154071a69fe7880ae - https://github.com/rancher/elemental-docs/pull/398
https://github.com/rancher/elemental-product-docs/pull/43/commits/edcbf99c5130623f05c9871883efbeb9096aa8b9 - https://github.com/rancher/elemental-docs/pull/413
https://github.com/rancher/elemental-product-docs/pull/43/commits/907b824e251519a52f36202222674c4c0476db54 - https://github.com/rancher/elemental-docs/pull/421
https://github.com/rancher/elemental-product-docs/pull/43/commits/36291d6809879a0edae55bc9cc5f69c1ce15f5d5 - https://github.com/rancher/elemental-docs/pull/425
https://github.com/rancher/elemental-product-docs/pull/43/commits/68b8df75d9061555c55d87f462b6d1ea83647237 - https://github.com/rancher/elemental-docs/pull/429
https://github.com/rancher/elemental-product-docs/pull/43/commits/d5af9169b2e80f24f079439f15cbda9b4ab6b347 - https://github.com/rancher/elemental-docs/pull/437
https://github.com/rancher/elemental-product-docs/pull/43/commits/fea37eb760827e285eecce49b043e597766b9dea - https://github.com/rancher/elemental-docs/pull/440
https://github.com/rancher/elemental-product-docs/pull/43/commits/e1466e944f4cd3a0c23fe1c9d00a733bcca819d6 - https://github.com/rancher/elemental-docs/pull/455
https://github.com/rancher/elemental-product-docs/pull/43/commits/cb1b58d6b00705c91863f9ccb4cb833d9423c50f - https://github.com/rancher/elemental-docs/pull/460
https://github.com/rancher/elemental-product-docs/pull/43/commits/64e9642520ca3cc0e5f65368c7d0734fe8b1743a - https://github.com/rancher/elemental-docs/pull/463

Updating Next/v1.7 related commits:
Reference commit - Reference PR:
https://github.com/rancher/elemental-product-docs/pull/43/commits/020e78fc5462a4235b2732d98cd87e2a8362856c - https://github.com/rancher/elemental-docs/pull/378
https://github.com/rancher/elemental-product-docs/pull/43/commits/1a5ecfe4f91fa42630bc51c66fc7311d39f18459 - https://github.com/rancher/elemental-docs/pull/407
https://github.com/rancher/elemental-product-docs/pull/43/commits/333a6952fa32b060aafd1f2c99cc40ba983928ce - https://github.com/rancher/elemental-docs/pull/447
https://github.com/rancher/elemental-product-docs/pull/43/commits/cb99f0ebdb7e9b7c4fd207052b560dab44c9903e (Syncing examples folder with community)
https://github.com/rancher/elemental-product-docs/pull/43/commits/dbfa133347ade7b85158a9cdc13840cda12c96c6 and https://github.com/rancher/elemental-product-docs/pull/43/commits/69e688943373f6f02bce0ee1cf85d33cd50a840b (Syncing v1.7 networking pages updates)

Follow-up: In a follow-up PR I will be re-organizing the Elemental /pages content folders and nav files to reflect the display organization seen in the left-hand side navigation in the DSC Elemental product site as opposed to the "flat" content structure currently seen. This is to align and standardize with product docs repos and improve content organization.

btat commented on docs/1.7/antora.yml:
```diff
@@ -0,0 +1,42 @@
+version: 1.7
+display_version: '1.7'
+start_page: en:index.adoc
+source-highlighter: highlight.js
```
```suggestion
```

This isn't needed since highlight.js is [already the default](https://docs.antora.org/antora/latest/asciidoc/source/#source-highlighter). It's also something that should be set at the site-level if customization is needed.

btat reviewed (changes requested):
Are all the excalidraw `SUSE® Rancher Prime OS Manager` -> `Elemental` changes reverting instances that were rebranded when they didn't need to be?

For the new versions/folders being introduced we probably want the entire version/folder to be craweled/indexed, so the revdates should all be updated.

Nit: most of the 1.5/* changes are strictly typos and don't need the revdate updated.

sunilarjun commented:
> Are all the excalidraw `SUSE® Rancher Prime OS Manager` -> `Elemental` changes reverting instances that were rebranded when they didn't need to be?
> 
> Nit: most of the 1.5/* changes are strictly typos and don't need the revdate updated.

Thanks for the review Billy! Didn't mean to revert those Excalidraw changes, reverted those in this commit for reference: https://github.com/rancher/elemental-product-docs/pull/43/commits/3fb3d5159ead552d9360e8345a50a79d66826938. Also removed the [highlighter syntax](https://github.com/rancher/elemental-product-docs/pull/43/commits/3f23f92991068eece91aa829e85ef30b569d4b35) and reverted the revdate changes for spelling only in these commits across versions ([v1.5](https://github.com/rancher/elemental-product-docs/pull/43/commits/61f71df6d438d3caa082d82b8eaed4ce7a368723) and [remaining versions](https://github.com/rancher/elemental-product-docs/pull/43/commits/4d7516c562fe88a311e1689628e43a75372e1d73)).

Please let me know if any more edits are needed, thanks!
